web/modules/custom/pineapple_mail/pineapple_mail.module

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_mail.module.
+ */
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_mail().
+ */
+function pineapple_mail_mail($key, &$message, $params) {
+  $options = array(
+    'langcode' => $message['langcode'],
+  );
+  switch ($key) {
+    case 'node_insert':
+      $message['from'] = \Drupal::config('system.site')->get('mail');
+      $message['subject'] = t('Subject: @title', array('@title' => $params['title']), $options);
+      $message['body'][] = Html::escape($params['message']);
+      break;
+  }
+}
+
+function pineapple_mail_ready_to_review(Drupal\Core\Entity\EntityInterface $entity) {
+  $mailManager = \Drupal::service('plugin.manager.mail');
+  $module = 'pineapple_mail';
+  $key = 'node_insert';
+  $to = \Drupal::config('system.site')->get('mail'); // site email
+  $entity_link = $entity->toUrl()->toString();
+  $entity_title = $entity->label();
+  $params['message'] = 'The user ' .\Drupal::currentUser()->getAccountName(). ' indicated that ' .$entity_title. ' is ready for review. View it here: www.easyrecipedelight.com' .$entity_link;
+  $params['title'] = $entity_title . ' is ready to review';
+  $langcode = \Drupal::currentUser()->getPreferredLangcode();
+  $send = true;
+
+  $result = $mailManager->mail($module, $key, $to, $langcode, $params, NULL, $send);
+  if ($result['result'] !== true) {
+    \Drupal::messenger()->addMessage('There was a problem sending your message and it was not sent.');
+  }
+  else {
+    \Drupal::messenger()->addMessage('Your message has been sent.');
+  }
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function pineapple_mail_node_presave(Drupal\Core\Entity\EntityInterface $entity) {
+  $ready_to_review = $entity->get('field_ready_to_review')->getString();
+  $published = $entity->get('status')->getValue()[0]['value'];
```

Comment:
I'd rename this variable to something else, because technically what it's getting is the publication status of the node.

Reply:
That works, thank you!